### PR TITLE
[FVM] beyond EVM part 6.2 - Implement EVM.BridgedAccount.withdraw

### PR DESCRIPTION
**Pull request by turbolent**
Depends on #4889

### CODE REVIEW

File: fvm/evm/stdlib/contract_test.go [hunk 1]
```diff
@@ -809,3 +815,101 @@ func TestEVMAddressDeposit(t *testing.T) {
...
+	rt := runtime.NewInterpreterRuntime(runtime.Config{})
+
+	script := []byte(`
+      import EVM from 0x1
+      import FlowToken from 0x1
+
+      access(all)
+      fun main(): UFix64 {
+          let bridgedAccount <- EVM.createBridgedAccount()
+          let vault <- bridgedAccount.withdraw(balance: EVM.Balance(flow: 0.0))
```
Comment (ramtinms) [inline fvm/evm/stdlib/contract_test.go:857]: does destroying a flow vault impact the total token count stored on the Flow token smart contract?

Comment (turbolent) [reply fvm/evm/stdlib/contract_test.go:857]: Yes! This is why the tests deposit/withdraw 0 for now. We still need to properly mint tokens in the test. When I didn't mint tokens and tried to depositing a non-zero amount, the total supply was underflowing :D

**Review by janezpodhostnik — APPROVED** [review]

**Comment by turbolent** [comment]
@ramtinms with `master` merged, `TestBridgedAccountWithdraw` fails with `evm runtime error: account does not exist`. Don't think that's caused by the PR itself. Any ideas?

**Comment by ramtinms** [comment]
@turbolent probably missing the flag for bootstraping, let me take a look

**Pull request merged**